lib/vanagon/platform/windows.rb

```diff
@@ -34,6 +34,30 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"
       end
 
+      # Add a repository (or install Chocolatey)
+      # Note - this only prepares the list of commands to be executed once the Platform
+      # has been setup
+      #
+      # @param definition [String] Definition for adding repo, can be a Repo URL (including file:)
+      #    If file suffix is 'ps1' it is downloaded and executed to install chocolatey
+      # @return [Array]  Commands to executed after platform startup
+      def add_repository(definition)
```

Comment:
`definition` is what all other `add_repository` methods use. It made sense to follow that established pattern.